src/services/pact-verifier-service.test.ts

```diff
@@ -37,40 +42,32 @@ describe('PactVerifierService', () => {
     }).compile();
 
     pactVerifierService = moduleRef.get<PactVerifierService>(PactVerifierService);
-    pactVerifierProvider = moduleRef.get(PactModuleProviders.PactVerifier);
+    pactVerifier = moduleRef.get<Verifier>(PactModuleProviders.PactVerifier);
+  });
+
+  beforeEach(() => {
+    jest.clearAllMocks();
   });
 
   describe('when configuring the providerBaseHost', () => {
     beforeEach(async () => {
-      options = { ...options, providerHost: '0.0.0.0' };
-      await pactVerifierService.verify(appMock as any);
+      options = { ...options, providerBaseUrl: 'http://127.0.0.1:80' };
+      await pactVerifierService.verify(appMock as unknown as INestApplication);
     });
     test('allows configuring hosts', async () => {
-      expect(appMock.listen).toHaveBeenCalledWith(80, '0.0.0.0');
+      expect(appMock.listen).toHaveBeenCalledWith(80, '127.0.0.1');
     });
-  });
-
-  describe("When calling the 'verify' method", () => {
-    beforeAll(async () => {
-      await pactVerifierService.verify(appMock as any);
-    });
-
-    test("then call 'verifyProvider' with the exact options", async () => {
-      expect(pactVerifierProvider.verifyProvider).toHaveBeenCalledWith({
-        ...options,
-        providerBaseUrl: '127.0.0.1:80',
-      });
+    test("then call 'verifyProvider' is called", async () => {
+      expect(pactVerifier.verifyProvider).toHaveBeenCalled();
     });
-
     test('then call the application methods', () => {
       expect(appMock.close).toHaveBeenCalled();
       expect(appMock.listen).toHaveBeenCalled();
     });
-  });
-
-  describe("When calling the 'getVerifier' method", () => {
-    test('then return something', () => {
-      expect(pactVerifierService.getVerifier()).toBeInstanceOf(VerifierMock);
+    describe("When calling the 'getVerifier' method", () => {
+      test('then return something', () => {
```

Comment:
?

Reply:
what do you mean with this comment?

Reply:
I think he might mean what does "then return something" mean - can this test name be more specific?

Reply:
Let's wait for your merge @TimothyJones - After that I will update the naming :)

Reply:
test is outdated as we removed the GetVerifier method (as we don't have the verifier provider anymore.